src/gui/gui_topbar.hpp

```diff
@@ -975,7 +975,7 @@ class topbar_speeddown_button : public button_element_base {
 public:
 	void on_create(sys::state& state) noexcept override {
 		button_element_base::on_create(state);
-		base_data.data.button.shortcut = sys::virtual_key::MINUS;
+		base_data.data.button.shortcut = sys::virtual_key::SUBTRACT;
```

Comment:
Instead of doing this, make one of the key codes mapped to the other when we send key through the ui. That way both keyboard keys can be used to adjust the speed (not all keyboards have a numpad)